lib/cocoapods/installer/analyzer.rb

```diff
@@ -22,6 +22,11 @@ class Analyzer
       #
       IOS_64_BIT_ONLY_VERSION = Version.new('11.0')
 
+      # @return [Integer] The Xcode object version until which 64-bit architectures should be manually specified
+      #
+      # Xcode 10 will automatically select the correct architectures based on deployment target
+      IOS_64_BIT_ONLY_PROJECT_VERSION = 50
```

Comment:
is 50 correct here? or did Xcode 10 start with 51?

Reply:
yes see `target_integrator.rb`

```ruby
          def input_output_paths_use_filelist?(object)
            object.project.object_version.to_i >= 50
          end
```

Can we use this constant there too?